src/test.util.tsx

```diff
@@ -0,0 +1,662 @@
+import { type FetchResponse } from '@openmrs/esm-framework';
+
+export const mockPUTResponse: FetchResponse<unknown> = {
+  data: {},
+  headers: new Headers(),
+  ok: true,
+  redirected: false,
+  status: 200,
+  statusText: 'OK',
+  url: 'http://mock.url',
+  type: 'error',
+  clone: function (): Response {
+    throw new Error('Function not implemented.');
+  },
+  body: undefined,
+  bodyUsed: false,
+  arrayBuffer: function (): Promise<ArrayBuffer> {
+    throw new Error('Function not implemented.');
+  },
+  blob: function (): Promise<Blob> {
+    throw new Error('Function not implemented.');
+  },
+  formData: function (): Promise<FormData> {
+    throw new Error('Function not implemented.');
+  },
+  json: function (): Promise<any> {
+    throw new Error('Function not implemented.');
+  },
+  text: function (): Promise<string> {
+    throw new Error('Function not implemented.');
+  },
+};
+
+export const mockIPS = {
```

Comment:
The usual convention would be to put this in a __mocks__ folder rather than directly in src.